corehq/apps/hqwebapp/utils/bootstrap/git.py

```diff
@@ -26,3 +26,14 @@ def has_no_pending_git_changes():
         ["git", "status"], stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE
     )
     return "nothing to commit" in str(status.communicate()[0])
+
+
+def ensure_no_pending_changes_before_continuing():
+    continue_message = "\nENTER to continue..."
+    while True:
```

Comment:
I'm not a fan of forcibly blocking the user - maybe they have unrelated changes in their repo - but they can cancel out of the command altogether, so this isn't blocking feedback.

Reply:
ok, it's only blocking for the initial use of the `skip-all` option